less/home-page.less

```diff
@@ -1,21 +1,58 @@
+
+
+
+
+
 .home {
+  padding: 0 40px;
 
+  .photoheader {
+    margin: 0 1%;
+    padding: 0px auto;
+    
+  }
+  .welcome {
```

Comment:
The `.welcome` element holds both the `h2` and `p` elements. Instead of giving the h2 top padding and the p bottom padding, why not just have:
```
.welcome {
    padding: 2% 0;
    border-bottom: 2px dashed @navigation-border;
}
```
This way, you don't even have to type styling for the h2 or p elements. Not a huge plus, but it makes the code a little bit cleaner to read '' '